src/tools/gophish_export.py

```diff
@@ -94,25 +116,30 @@ def pull_gophish_groups(api, assessment_id):
 
 
 def campaignControl(api, assessment_id):
-    """Control the campaign importing of an assessment to DB."""
-    # STAND ALONE
+    """Control the campaign importing of an assessment to DB.
+
+    Results are saved to a json file.
+
+    Args:
+        api (GoPhish API): Connection to GoPhish server via the API.
+        assessment_id (string): Assessment identifier to get campaigns from.
+
+    Returns:
+        List of the assessment's campaigns with data.
+    """
     campaignIDs = pull_gophish_campaign(api, assessment_id)
     campaigns = list()
 
     for campaign_id in campaignIDs:
         campaigns.append(import_campaign(api, campaign_id))
 
-    logging.info("Successfully added campaigns for {}".format(assessment_id))
+    logging.info(f"Successfully added campaigns for {assessment_id}")
 
-    with open("data_" + assessment_id + ".json", "a") as fp:
-        json.dump(campaigns, fp, indent=4)
-
-    return True
+    return campaigns
 
 
 def pull_gophish_campaign(api, assessment_id):
```

Comment:
I would recommend renaming this function `get_campaign_ids`.

Reply:
Changed name in fefb0b4.